core/src/main/scala/com/codacy/analysis/core/utils/HttpHelper.scala

```diff
@@ -2,9 +2,9 @@ package com.codacy.analysis.core.utils
 
 import io.circe.parser.parse
 import io.circe.{Json, ParsingFailure}
-import scalaj.http.{Http, HttpRequest, HttpResponse}
+import scalaj.http.{Http, HttpRequest, HttpResponse, HttpOptions}
 
-class HttpHelper(apiUrl: String, extraHeaders: Map[String, String]) {
+class HttpHelper(apiUrl: String, extraHeaders: Map[String, String], allowUnsafeSSL: Boolean = false) {
```

Comment:
There shouldn't be any default here since the default is already stated in the command line parser [here](https://github.com/codacy/codacy-analysis-cli/pull/413/files#diff-0938737d69935ac3c77b4ec16efbe0d99ea8c175052ed5c3c048973289cbbef5R150)